app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="app_name">Auditor</string>
+    <string name="url">attestation.app</string>
```

Comment:
Probably better to call this `base_url` and it can include the `https://` prefix.